var/spack/repos/builtin/packages/py-biobb-common/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyBiobbCommon(PythonPackage):
+    """Biobb_common is the base package required to use the biobb packages"""
+
+    pypi = "biobb_common/biobb_common-4.1.0.tar.gz"
+
+    maintainers("d-beltran")
+
+    # Versions
+    version("4.1.0", sha256="97637f359a3bb8ad79aca72b6c26f73fe2424845dc7f43005643971046e9d117")
+
+    # Dependencies
+    depends_on("py-setuptools", type="build")
+    depends_on("python@3.8:3.10", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("python@3.8:", type=("build", "run"))
```
I don't see any evidence that 3.11 doesn't work, do you?

Reply:
Acordint to [the docs](https://mmb.irbbarcelona.org/biobb/availability/python-compatibility) python 3.11 is not maintained

Reply:
Those are just the versions that are tested. It's a pure-Python library, so it's unlikely that the minor changes between Python releases would break something. Unless you know it doesn't work with 3.11, I would suggest the above change. But you're maintaining this recipe, so it's your final call.